astro/src/content/articles/identity-basics/what-is-oidc.mdx

```diff
@@ -0,0 +1,170 @@
+---
+title: "What Is OIDC?"
+description: "What Is OIDC, OpenID Connnect, And How Do You Use It?"
+author: Dan Moore
+icon: /img/icons/what-is-scim.svg
+section: Identity Basics
+date: 2023-12-10
+dateModified: 2023-12-10
+---
+import Aside from '/src/components/Aside.astro';
+import ExampleIdToken from 'src/content/docs/lifecycle/authenticate-users/oauth/_example_id_token.mdx';
+import OIDCFlow from "../../../diagrams/articles/identity-basics/oidc-flow.astro";
+
+
+OIDC, which stands for OpenID Connect, is a specification to allow users to authenticate using a standard protocol. This article covers what OIDC is, why you might want to use it, and how it works.
+
+## What Is OIDC?
+
+OIDC is a standard built on top of two different standards that solves the common problem of authenticating users. OIDC leverages both OAuth, which defines the data flow and specifics around token generation, and OpenID, which defines the authentication process.
+
+It's an extremely flexible framework that can cover both simple authentication systems and complex large-scale systems. There is a core set of functionality defined in the specification as well as a fair amount of optional behavior and additional specifications handling use cases beyond authentication such as sign out and session management.
+
+## What Does OIDC Allow?
+
+At a high level, using OIDC lets you delegate user authentication in a standardized fashion. By running an OIDC compliant identity data store, other applications, including custom built apps, open source applications, commercial off the shelf apps, and SaaS solutions, can all use the same authentication solution.
+
+For example, you could have your own application (custom-built), forum software like Discourse (open source), Oracle (COTS), and Zendesk all delegate authentication to your OIDC compliant user identity datastore. You can support both employees and customers using such a solution. In fact, because so many customer applications are web apps, OIDC is a perfect fit for a customer identity and access management (CIAM) system.
```

Comment:
```suggestion
For example, you could have your application (custom-built), forum software like Discourse (open source), Oracle (COTS), and Zendesk all delegate authentication to your OIDC-compliant user identity datastore. You can support both employees and customers using such a solution. Because many customer applications are web apps, OIDC is a perfect fit for a customer identity and access management (CIAM) system.
```